src/component/AddItem.js

```diff
@@ -46,6 +52,46 @@ const AddItem = ({ firestore }) => {
           className="inputField"
         />
       </label>
+
+      <div className="frequencyButtons">
+        <p>How soon are you likely to buy it again?</p>
+        <input
+          type="radio"
+          id="soonButton"
+          name="frequencyButtons"
+          value="soon"
+          checked={frequency === "soon"}
```

Comment:
Is this needed? I think you're already changing the frequency state in handleOptionChange on line 32

Reply:
This line is actually checking state, not changing it :) Just changing the way it looks, really!